src/IssueViz.Security/Taint/TaintResources.resx

```diff
@@ -135,4 +135,8 @@
   <data name="SyncPackage_Initializing" xml:space="preserve">
     <value>[Taint] Initializing taint issues synchronization package...</value>
   </data>
+  <data name="Synchronizer_UnsupportedSQVersion" xml:space="preserve">
+    <value>[Taint] Cannot fetch taint vulnerabilities: requires SonarQube v8.6 or later, or SonarCloud. Connected SonarQube version: v{0}
```

Comment:
`Cannot fetch` makes it sound a bit like an exception, perhaps it's worth phrasing it in a more neutral manner, i.e. 

`[Taint] Taint Vulnerabilities are not supported in the connected SonarQube server version (v{0}). Visit https://github.com/SonarSource/sonarlint-visualstudio/wiki to find out more about this and other SonarLint features.`

Reply:
Originally I had `Displaying TVs in the IDE requires...`. I changed the working to be consistent with the other condition checks i.e. standalone mode and not yet connected.

Reply:
Hm, I see your point. Perhaps it's worth changing them too to something "less scary"; I'd change this anyway though as it could be a marketing opportunity so it's worth making it sound attractive.